package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bottery",
+  "version": "1.0.0",
+  "description": "## A conversational agent prototyping platform by [Kate Compton](https://github.com/galaxykate).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/smishr4/bottery.git"
```

Comment:
Noticed @smishr4's username here. Should be google?

Reply:
Oh well, I cloned it from my fork thats why it had my name, I think you are right I should fix it. Let me know if you are sure about that.

Reply:
fixed in the below commit 👍